client/tailwind.config.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+  theme: {
+    extend: {
+      fontFamily: {
+        proxima: 'Proxima Nova'
+      },
+      colors: {
+        primaryBlue: '#006C9E',
+        blue2: '#004A6E',
+        blue3: '#E2EBF3',
+        blue4: '#F3F8FA',
+        gray1: '#333333',
+        gray2: '#4F4F4F',
+        gray3: '#828282',
+        lightGray: '#979797',
+        red1: '#D64B3A',
+        red2: '#F9E4E1',
+        orange1: '#F8921F',
+        orange2: '#FAE7D1',
+        green1: '#179D57',
+        green2: '#DCFAEA',
+        blueOverlay: 'rgba(0,74,110,0.5)'
+      }
+    }
+  },
+  plugins: [
+    // components
+    require('./tailwind/plugins/components/required')()
```

Comment:
:+1: 

'required' could be `../tailwind/plugins/utilities/required`

Reply:
Even though I'm using `addComponent`?  How can I tell what should be a component and what should be a utility?